### PR TITLE
Westleyk template new

### PR DESCRIPTION
**Pull request by WestleyK**
Finally! a good PR! 😉

### CODE REVIEW

**Comment by WestleyK** [comment]
A lot of commits! but it should do the trick.

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+## Which Document page:
+
+What page is the problem on.
```
Comment (GodloveD) [inline .github/issue_template.md]: On what page is the problem?

File: .github/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+### Expected results:
+
+What are you expexted to see.
```
Comment (GodloveD) [inline .github/issue_template.md]: expexted to expecting

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+
+### Actual results:
+
+What the problem is, whats misspelled, or whats missing.
```
Comment (GodloveD) [inline .github/issue_template.md]: What is the problem, what's misspelled, or what's missing?

File: .github/issue_template.md (hunk 2 above)
Comment (GodloveD) [reply .github/issue_template.md]: Also change the period to a question mark.

**Review by GodloveD — CHANGES_REQUESTED** [review]
few changes requested. overall this is a fine idea.

**Review by GodloveD — APPROVED** [review]
lgtm

**Comment by GodloveD** [comment]
:shipit:

**Comment by GodloveD** [comment]
Oh dern-it.  I just realized this PR isn't going to do a bit of good on the 3.0 branch.  It needs to go to master afaik for this to actually be displayed when somebody creates a pull request. Mind changing the base branch @WestleyK ?

**Comment by jchavezlavalle** [comment]
@GodloveD @WestleyK it's okay to be on 3.0, the workflow is: we're accepting the PRs on the 3.0 branch and then when it's live on the site, the master branch is updated to the latest version (so to 3.0 in this case) I think that I will add this on the original readme for the repo, so that it is clear for contributors that they need to send PRs to 3.0 branch :+1:

**Pull request merged**